src/openrct2-ui/UiContext.macOS.mm

```diff
@@ -118,14 +118,14 @@ void ShowMessageBox(SDL_Window * window, const std::string &message) override
                 panel.canChooseFiles = false;
                 panel.canChooseDirectories = true;
                 panel.allowsMultipleSelection = false;
-                utf8 *url = NULL;
                 if ([panel runModal] == NSFileHandlingPanelOKButton)
                 {
                     NSString *selectedPath = panel.URL.path;
                     const char *path = selectedPath.UTF8String;
```

Comment:
That is what this PR is doing, the conversion to `std::string` is implicit. Before it was duplicating the string unnecessary, because it gets duplicated anyway to convert it to the `std::string`.